draft-btw-add-home.txt

```diff
@@ -988,26 +989,25 @@ Internet-Draft       Encrypted DNS in Home Networks        November 2020
 
    o  Locator: 2001:db8:1::1
 
-   The client requests "https://doh.example.com/.well-known/resinfo" to
-   retrieve the list of the URI Templates associated with this
-   discovered server.
+   The client will use DEER [I-D.pauly-add-deer] to discover the DoH
```

Comment:
Idem as above. I think the ref should be updated to schwartz-svcb-dns.

Reply:
See other thread; I've added some references. Having said that, I'm open to rephrasing to make it clear that DEER means using SVCB queries.